torch/distributed/tensor/parallel/style.py

```diff
@@ -66,7 +65,7 @@ def __init__(self, _prepare_input=None, _prepare_output=None) -> None:
         super().__init__(_prepare_input, _prepare_output)
 
 
-class PairwiseSequenceParallel(PairwiseParallel):
+class SequenceParallel(PairwiseParallel):
     """
     PairwiseSequenceParallel concatenate colwise and rowwise styles as a fixed
```

Comment:
can we also fix the docs so to the correct name?

Reply:
oh sure.